### ISSUE TITLE
cover.html : impossible to display image

### ISSUE THREAD

**Issue by impactJL** [splitbrain/dokuwiki-plugin-dw2pdf#221]
Hello
i'm using dw2pdf and i would like to customize the cover. cover.html works pretty well.
I have a issue but i think i'm using the wrong way : 
i would like to display a picture on this cover. the picture is in the same repartory than the html file.

I tried with a "classic" html command : `<p style="text-align:center"><img src="logo.jpg" alt="dfdfdfdfdf" width="500"></p>`
it doesn(t work, the image was not display, and then alt text "dfdfdfdfdf"  too

i also try the same with the full path, and same result

i guess i shoudl use @TPLBASE@ but i do not know how to use it :  @TPLBASE@logo.jpeg ? 

thank in advance to help me
Regards

**Comment by impactJL** [comment]
hello
i have checked again the full path and it works
i guess the syntaxe was wrong
anyway i would be pleased to get information about TPLBASE
regards

**Comment by nerfer** [comment]
I have tried the full path as well, and it initially didn't work for me: `<img src="http://www..../data/media/logo.jpg">`
I also tried some relative paths, but I'm not entirely sure where it would be starting from, and that didn't work either. When I entered that same full address into the browser, it said access was forbidden, which was unexpected to me.  Must be a permissions thing. 
So I copied that image to somewhere else on my site and then it worked.
